File: SistemaGestaoQualidade.WebApi/SistemaGestaoQualidade.WebApi/Models/NaoConformidade.cs
using NaoConformidadeModule.WebApi.Shared;
using System;

namespace NaoConformidadeModule.WebApi.Models
{
    public class NaoConformidade : IIdentityEntity
    {
        public int id { get; set; }
        public string dataNC { get; set; }
        public string emitenteNC { get; set; }
        public int idCausaNC { get; set; }
        public int idEstadoNC { get; set; }
        public int idIdentificacaoNC {get;set;}
        public int idIntensidadeNC { get; set; }
        public int idRequisitoNC { get; set; }
        public int idTipoRequisito { get; set; }
        public string descricaoNC { get; set; }
        public string abrangenciaNC { get; set; }        
        public string acaoImediata { get; set; }        
        public string dataAcaoImediata { get; set; }
        public string responsavelAcaoImediata { get; set; }
        public Boolean temAnaliseCR { get; set; }
        public Boolean cancelouNC { get; set; }
    }
}

File: sistema-gestao-qualidade-poc/src/app/Models/nao-conformidade.ts
export class NaoConformidade{
          constructor(
            public id: number,
            public dataNC: string,
            public emitenteNC: string,
            public idCausaNC: number,
            public idEstadoNC: number,
            public idIdentificacaoNC: number,
            public idIntensidadeNC: number,
            public idRequisitoNC: number,
            public idTipoRequisito: number,
            public descricaoNC: string,
            public abrangenciaNC: string,
            public acaoImediata: string,
            public dataAcaoImediata: string,
            public responsavelAcaoImediata: string,
            public temAnaliseCR: boolean,
            public cancelouNC: boolean
          ){}
      
      }